XamlControlsGallery/ControlPages/RichEditBoxPage.xaml.cs

```diff
@@ -9,6 +9,7 @@
 //*********************************************************
 using System;
 using System.Collections.Generic;
+using System.Diagnostics;
```

Comment:
What is this include used for?

Reply:
Thanks for pointing that out! It is not necessary and I removed it.